INSTALL.md

```diff
@@ -1,27 +1,26 @@
-## Installation (with docker-compose)
+## Installation (with docker compose)
 
-To simply try out mampf you can use `docker-compose` ([needs docker](https://docs.docker.com/engine/install/ubuntu/) && `apt install docker-compose`). Simply clone the mampf repository and run docker-compose by executing
+To simply try out mampf you can use `docker compose` (needs [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/)). Simply clone the MaMpf repository and run docker compose by executing
 ```
-$ git clone -b main --recursive https://github.com/MaMpf-HD/mampf.git
-
+$ git clone -b main --recursive git@github.com:MaMpf-HD/mampf.git
```

Comment:
I dont know whether this is an actual improvement, as git over http requires no configuration and is therefore easily doable. Git over ssh has the advantage that later development is more easily doable without changing the git upstream url.

Reply:
True, but right now we have ssh in the Readme and http in the Install markdown. Will go with http as no configuration is required ;)